website/blog/2025-12-10-react-native-0.83.md

```diff
@@ -0,0 +1,272 @@
+
+:::info
+`RCT_REMOVE_LEGACY_ARCH` is not compatible with React Native precompiled binaries. If you are using precompiled binaries, you'll need to reinstall the pods and build the app from source.
+:::
+
+### Experimental - Debug your precompiled binaries on iOS
+
+In this release, we've implemented the ability to debug the React Native code that is shipped with a precompiled binary. This is primarily helpful to library maintainers or if you are developing a native module or a native component.
+
+To be able to debug the binary code shipped with the React Native prebuild, follow these steps:
+
+```sh
+# From the ios folder of your project
+bundle exec pod cache clean --all
+bundle exec pod deintegrate
+RCT_USE_RN_DEP=1 RCT_USE_PREBUILT_RNCORE=1
+RCT_SYMBOLICATE_PREBUILT_FRAMEWORKS=1 bundle exec pod install
+open <your-project>.xcworkspace
+```
+
+The magic is done by the `RCT_SYMBOLICATE_PREBUILT_FRAMEWORKS` flag, which instructs CocoaPods to download the React Native dSYM files and expand them in the proper folder.
+
+At this point, you can put a breakpoint in your app, for example in `AppDelegate.swift`, and build and run the app from Xcode.
+
+When the application pauses, open the Xcode console and run the LLDB command:
+
+```sh
+# If you are running the app in the simulator
+add-dsym <path-to-your-app>/ios/Pods/React-Core-prebuilt/React.xcframework/ios-arm64_x86_64-simulator/React.framework/dSYMs/React.framework.dSYM
+
+# If you are running the app on a physical device
+add-dsym <path-to-your-app>/ios/Pods/React-Core-prebuilt/React.xcframework/ios-arm64/React.framework/dSYMs/React.framework.dSYM
+```
+
+Now you can step into the React Native code.
+
+## Breaking Changes
+
+We're working hard to make React Native releases more predictable and easier to upgrade. React Native 0.83 is the first release with no user facing breaking changes. If you are on React Native 0.82, you should be able to upgrade your app to React Native 0.83 without any changes to your app code.
+
+To learn more about what we consider a breaking change, have a look at [this article](link-to-breaking-changes).
+
+## Deprecations
+
+This release ships two deprecations that are Android specific:
+
+- **Networking**: The `sendRequestInternal` method is being phased out and it is now deprecated-
+- **Animation**: `startOperationBatch` and `finishOperationBatch` are now deprecated.
+
+## Acknowledgements
+
+React Native 0.83 contains over 594 commits from 56 contributors. Thanks for all your hard work!
+
+We want to send a special thank you to those community members that shipped significant contributions in this release:
+
+- [Ruslan Lesiutin](https://github.com/hoxyq), [Moti Zilberman](https://github.com/motiz88), and [Alex Hunt](https://x.com/huntie) for the React Native DevTools Performance and Network features.
```

Comment:
```suggestion
- [Ruslan Lesiutin](https://github.com/hoxyq), and [Moti Zilberman](https://github.com/motiz88) for the React Native DevTools Performance and Network features.
```
I know that you worked on the feature, but you are also the author of the blogpost and it is implied that you have been working on these things. We never add the blogpost authors to the acknowledgements, usually.